packages/mambajs-core/src/helper.ts

```diff
@@ -216,6 +221,8 @@ export function removeFilesFromEmscriptenFS(FS: any, paths: any): void {
         } else {
           FS.unlink(path);
         }
+      } else {
+        logger?.log(`Path ${path} does not exist`);
```

Comment:
I think we should only log in the console instead, since the logger is for the end-user

```suggestion
      } else {
        console.log(`Uninstall error: Path ${path} does not exist`);
```